instant-distance-py/src/lib.rs

```diff
@@ -79,6 +80,63 @@ impl Hnsw {
     }
 }
 
+#[pyclass]
+struct HnswMap {
+    inner: instant_distance::HnswMap<FloatArray, String>,
+}
+
+#[pymethods]
+impl HnswMap {
+    #[getter]
+    fn values(&self) -> PyResult<Vec<String>> {
+        Ok(self.inner.values.clone())
+    }
+    /// Build the index
+    #[staticmethod]
+    fn build(points: &PyList, values: Vec<String>, config: &Config) -> PyResult<Self> {
```

Comment:
`values: Vec<String>` uses Rust library instead of Python-native (like `PyList`). There are pros/cons: https://pyo3.rs/master/conversions/tables.html#using-rust-library-types-vs-python-native-types